### ISSUE TITLE
Add team's pictures and social media links

### ISSUE THREAD

**Issue by Neha** [jslovers/jslovers-branding-page#9]
Add the team's pictures and social media links

**Comment by ksubramanyeshwara** [comment]
Do you have a particular design in your mind?

**Comment by Neha** [comment]
> Any particular design is in your mind?

Thank you @ksubramanyeshwara. I am thinking, we can have the same design as we have of sponsors section

**Comment by sachin9164** [comment]
Can i pick this?

**Comment by Neha** [comment]
Let's wait for @ksubramanyeshwara as he queried first. If he won't picked this issue then I will assign it to you

**Comment by ksubramanyeshwara** [comment]
@Neha Do we have a specific timeframe?

**Comment by Neha** [comment]
@ksubramanyeshwara 1 week

**Comment by ksubramanyeshwara** [comment]
@Neha Can I use font awesome for social media icons?

**Comment by Neha** [comment]
@ksubramanyeshwara sure. Font awesome will work. 

If you are going to pick this issue...can I assign it to you?

**Comment by ksubramanyeshwara** [comment]
@Neha Please do. I am already working. Thank you

**Comment by Neha** [comment]
Thank you @ksubramanyeshwara . This looks amazing. 

A small suggestion: Instead of green colour can we have a grey or blue colour, please?

For social media: Please keep LinkedIn, Twitter, and GitHub

**Comment by ksubramanyeshwara** [comment]
@Neha Absolutely! We have the option of either grey or blue. I'll change the one you suggest.
I'll remove other social media handles.

**Comment by ksubramanyeshwara** [comment]
@Neha Please ensure all the social media handles are up-to-date so that I can proceed with the update.

**Comment by PreetiW** [comment]
Pune:

Preeti
LinkedIn - https://www.linkedin.com/in/preetiwadhwani/
Github - https://github.com/PreetiW
Twitter - https://twitter.com/preetiwadhwani_
Profile Pic - https://twitter.com/preetiwadhwani_/photo

**Comment by ksubramanyeshwara** [comment]
@Neha can you please update everyone's images and social media account so that i can complete it. Thank you
